Do-ItCore/Organization/NotificationManager.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  NotificationManager.swift
+//  Do-ItCore
+//
+//  Created by Cesar F. Chacon on 2/27/19.
+//  Copyright © 2019 The Swifter Picker-Uppers. All rights reserved.
+//
+
+import UserNotifications
+
+class NotificationManager {
+    let center = UNUserNotificationCenter.current()
+
+    func requestAuthorization() {
+        center.requestAuthorization(options: [.alert, .sound]) {
+            (granted, error) in
+                // do something if granted is false, user did not allow notifications
```

Comment:
For now, you can just print the response to the console. If they reenable notifications later, the `setTrigger` function will start working fine.